core/domain/prod_validation_jobs_one_off.py

```diff
@@ -321,39 +380,53 @@ def _get_external_model_properties(cls):
         This should be implemented by subclasses.
 
         Returns:
-            tuple(str, list(tuple), dict). A tuple with first element as
-                external model name, second element as a tuple of
-                cls.external_instance_details and the third element
-                as a properties dict with key as property name in summary
-                model and value as property name in external model.
+            tuple(str, str, dict). A tuple with first element as
+                external model name, second element as a key to fetch
+                external model details from cls.external_instance_details
+                and the third element as a properties dict with key as
+                property name in summary model and value as property name
+                in external model.
 
         Raises:
             NotImplementedError. This function has not yet been implemented.
         """
         raise NotImplementedError
 
     @classmethod
-    def _validate_external_model_properties(cls, item):
+    def _validate_external_model_properties(
+            cls, item, external_instance_details):
         """Validate that properties of the model match the corresponding
         properties of the external model.
 
         Args:
             item: ndb.Model. BaseSummaryModel to validate.
+            external_instance_details: dict(str, (list(str, str, ndb.Model))).
```

Comment:
Should the typeinfo and description be changed?

Reply:
Yes, updated.